web/src/app/services/hamburger/hamburger.service.ts

```diff
@@ -0,0 +1,20 @@
+import {Injectable} from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class HamburgerService {
```

Comment:
"Hamburger" is just the name of the icon. Let's call this something more self-describing? Or even better, should this be part of a broader service for global nav, etc?